### ISSUE TITLE
[PAW] PACT Act content updates

### ISSUE THREAD

**Issue by FranECross** [department-of-veterans-affairs/va.gov-cms#16128]
## Status
**[2024-07-18--23] [Fran]** Pulling out the new results logic and screen, and new ticket [created](https://app.zenhub.com/workspaces/vagov-sitewide-product-team-6645154188a81a1f7a0a9464/issues/gh/department-of-veterans-affairs/va.gov-cms/18720) . We're awaiting the content from Danielle T.
**[2024-07-18] [Fran]** Jordan finished her work; moved ticket back to Ready; Fran to update AC; Michelle following up with Danielle regarding content for the final screen. (Ticket points adjusted to a 3, which will be for engineering.)
**[2024-07-10]** Mural for visual of new/changed content provided by Danielle
**[2024-07-09]** Michelle has confirmed with Danielle that we are expecting an additional review screen for the PACT Act Wizard, in addition to the two additional questions. She'll send a visual with the new screens by EOD or first thing tomorrow
**[2024-06-27]** Information provided on anticipated changes (see Description below) - to be confirmed by Danielle prior to Sprint 8 start.
**[2024-05-07]** No updates from stakeholders yet.

## Description
Update the PACT Act logic flow with updated content provided by Danielle T. 

**_Note that this ticket encompasses the UX Design work, and then it will be turned over to engineering for the actual implementation._**

  - New question in Burn Pit flow S2.1 
  - New question in Main flow S2.5, asked for all Veterans, regardless of the option they select for S1.
 
(Note: Logic change and results screen pulled out; see status at top of ticket)

### User story

**AS A** Veteran or other PAW user
**I WANT** to see updated content in the PAW
**SO THAT** I answer the questions without confusion as I complete the PAW

### Change details (provided 2024-07-10) 
- New question in Burn Pit flow S2.1 
- New question in Main flow S2.5, asked for all Veterans, regardless of the option they select for S1.
(Note: Logic change and results screen pulled out; see status at top of ticket)

### References for details
- [PACT Act wizard_updated with health care expansion changes_2024-07-10_15-43-13.pdf](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNWJMQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--98ec3f247d75278fab0917c4d7d980da9cca2621/PACT%20Act%20wizard_updated%20with%20health%20care%20expansion%20changes_2024-07-10_15-43-13.pdf)
- [Or link to Mural with updated health care expansion changes](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1720458778226/639e2221f25b7a2580e3904f8087b321cfacab9a?sender=u3fcece2648da0fcf15d81890)

## UX/Design Considerations (2 points)
Per AC below, the changes will encompass the following:
- [x] New question in Burn Pit flow S2.1 
- [x] New question in Main flow S2.5, asked for all Veterans, regardless of the option they select for S1.
- [x]  New results screen shown if Veteran selects "Yes" for S2.1 and/or S2.5 only.
- [x]  Logic change for results screens

## Engineering Acceptance Criteria (3 points)
- [ ] New question in Burn Pit flow S2.1 
- [ ] New question in Main flow S2.5, asked for all Veterans, regardless of the option they select for S1.
- [ ] a11y Review

(Note: Logic change and results screen pulled out; see status at top of ticket)

**Comment by cindymerrill** [comment]
The above are NOT functional changes from Danielle but instead ways that her Word doc was out of date from what's in Survey Monkey and the [interaction flow diagram](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1692989444688/0044b9825c82d8d23920601f68c41a61d047d681?sender=uff8fa85716c1796ccf022668). Danielle updated her Word document to align. FYI @jilladams

**Comment by FranECross** [comment]
Blocked until final content comes through. Jordan will need to update the Sketch file.

**Comment by mmiddaugh** [comment]
@FranECross description has been updated with the information about anticipated changes. Danielle will confirm these changes before the start of our Sprint 8.

**Comment by thejordanwood** [comment]
- @randimays I've updated our [PACT Act Mural](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1692989444688/0044b9825c82d8d23920601f68c41a61d047d681?sender=u01810f3edbfc878c08be5925) to show the new changes. 
- @FranECross I have the link to the [outdated Sharepoint doc](https://dvagov-my.sharepoint.com/:w:/r/personal/danielle_thierry_va_gov/Documents/PACT%20Act%20wizard%20content%20for%20usability%20testing.docx?d=wfee483299a674f38ac4ec198c9cca463&csf=1&web=1&e=66EcOc) that has all of the copy for this. Do you have the new copy that shows what should be on the additional results screen? I can add that to the Mural.

**Comment by randimays** [comment]
Update: this will carry over to the next sprint. I am also waiting on Danielle's content for review page 4. @FranECross @mmiddaugh has that content been provided already?

**Comment by FranECross** [comment]
Logic and results screen pulled out of this ticket (content not available yet) and a new [ticket ](https://app.zenhub.com/workspaces/vagov-sitewide-product-team-6645154188a81a1f7a0a9464/issues/gh/department-of-veterans-affairs/va.gov-cms/18720)has been created.

**Comment by randimays** [comment]
This has been merged and verified in staging. Content for results page 4 and updated content for results pages 1.1-1.2 will be added in #18720. Closing as complete.
